packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -841,6 +841,38 @@ pub async fn remove_indexer(
     Ok(())
 }
 
+pub async fn remove_previous_indexer(
```

Comment:
Can we be a little more specific with the naming? There can be multiple "previous indexers". This function seems to remove the most recent version of the indexer. Can we update the naming to reflect that?